packages/tests/server/regression/issue-5034-input-with-index-signature.test.ts

```diff
@@ -60,4 +74,14 @@ describe('inferRouterInputs/inferRouterOutputs', () => {
     expectTypeOf<Input>().toEqualTypeOf<{ name: string }>();
     expectTypeOf<Output>().toEqualTypeOf<{ name: string }>();
   });
+
+  test('middleware with symbol key', async () => {
+    type Input = AppRouterInputs['middlewareWithSymbolKey'];
+    type Output = AppRouterOutputs['middlewareWithSymbolKey'];
+    expectTypeOf<Input>().toEqualTypeOf<{ name: string }>();
+    expectTypeOf<Output>().toEqualTypeOf<{
+      name: string;
+      [symbol]: true;
```

Comment:
Having a hard time with this... why are we expecting inferred outputs to contain symbols (which is not JSON serializable)? Plus, the `input` (which the query returns) doesn't contain the symbol, `ctx` does. And if you look at `ctx` inside the query, it's inferred correctly

Reply:
I messed up the test, it actually works as intended, sorry :D